irispy/sji.py

```diff
@@ -0,0 +1,188 @@
+import datetime
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+import sunpy.cm as cm
+import sunpy.instr.iris
+import sunpy.map
+import numpy as np
+import matplotlib.colors as colors
+from astropy.io import fits
+'''
+This module provides movie tools for level 2 IRIS SJI fits file
+'''
+
+
+def sji_fits_to_cube(filelist, start=0, stop=None, skip=None, grid=False):
+    """
+    Read SJI files and return a MapCube. Inherits
+    sunpy.instr.iris.SJI_to_cube to stitch multiple sji fits
+    files stitched into one mapcube.
+    Also sets colormap and normalization
+    Assumes hdu index of 0.
+
+
+    Parameters
+    ----------
+    filelist: string or list
+        File to read, if single file string detected, will
+        create a list of length 1.
+
+    start: step
```

Comment:
To add a little more technical to this when writing documentation in this format the right hand side of the colon should normally be a Python type, and should be in single back ticks. Assuming start, stop and skip here are supposed to be integers it should look like this:

```
start: `int`
```